psi4/src/psi4/ccdensity/ccdensity.cc

```diff
@@ -383,72 +383,29 @@ PsiReturnType ccdensity(std::shared_ptr<Wavefunction> ref_wfn, Options& options)
       }
       mo_offset += nmo;
     }
-    /*Call OEProp for each root opdm */
-    auto oe = std::make_shared<OEProp>(ref_wfn);
+
+    /* Transform Da/b to so basis and set in wfn */
     if(ref_wfn->same_a_b_dens()){
       Pa->scale(0.5);
-      oe->set_Da_mo(Pa);
-      Pb = Pa;
-    }else{
-      oe->set_Da_mo(Pa);
-      oe->set_Db_mo(Pb);
-    }
-    oe->add("DIPOLE");
-    oe->add("QUADRUPOLE");
-    oe->add("MULLIKEN_CHARGES");
-    oe->add("NO_OCCUPATIONS");
-    std::string cc_prop_label("CC");
-    if( i == 0 ){ // ground state
-      oe->set_title(cc_prop_label);
-      oe->compute();
-      //set OPDM in ref_wfn for GS only
-      SharedMatrix cc_Da = oe->Da_so();
-      SharedMatrix ref_Da = ref_wfn->Da();
-      ref_Da->copy(cc_Da);
-      if(params.nstates > 1){
-        Process::environment.globals["CC ROOT 0 DIPOLE X"] = Process::environment.globals["CC DIPOLE X"];
-        Process::environment.globals["CC ROOT 0 DIPOLE Y"] = Process::environment.globals["CC DIPOLE Y"];
-        Process::environment.globals["CC ROOT 0 DIPOLE Z"] = Process::environment.globals["CC DIPOLE Z"];
-        Process::environment.globals["CC ROOT 0 QUADRUPOLE XX"] = Process::environment.globals["CC QUADRUPOLE XX"];
-        Process::environment.globals["CC ROOT 0 QUADRUPOLE XY"] = Process::environment.globals["CC QUADRUPOLE XY"];
-        Process::environment.globals["CC ROOT 0 QUADRUPOLE XZ"] = Process::environment.globals["CC QUADRUPOLE XZ"];
-        Process::environment.globals["CC ROOT 0 QUADRUPOLE YY"] = Process::environment.globals["CC QUADRUPOLE YY"];
-        Process::environment.globals["CC ROOT 0 QUADRUPOLE YZ"] = Process::environment.globals["CC QUADRUPOLE YZ"];
-        Process::environment.globals["CC ROOT 0 QUADRUPOLE ZZ"] = Process::environment.globals["CC QUADRUPOLE ZZ"];
```

Comment:
Just leave the P::e block here, only commented. psivar scraper for docs is so dumb it doesn't notice.